src/services/taskratchet/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const HOSTNAME = window && window.location && window.location.hostname;
+export const IS_PRODUCTION = HOSTNAME === 'app.taskratchet.com';
+export const IS_STAGING = HOSTNAME && HOSTNAME.includes('deploy-preview');
```

Comment:
Can we add a todo comment to make sure we revisit these constants at some point?

Reply:
Added